docs/reference/announcements/880.md

```diff
@@ -137,25 +137,20 @@ Additional upgrade considerations are necessary for deployments that use custom
 
 ## Camunda Exporter <span class="badge badge--long" title="This feature affects Self-Managed">Self-Managed</span>
 
-A new Camunda Exporter brings the importer and archiving logic of web components (Tasklist and Operate) closer to the distributed platform (Zeebe). The index schema is also being harmonized.
+Our web applications have historically used importers and archivers to consume, aggregate, and archive historical data provided by our Elasticsearch (ES) or OpenSearch (OS) Exporter.
 
-The exporter can consume Zeebe records (mostly events created by the engine), aggregate data, and store the related data into shared and harmonized indices.
+![87-orchestration-cluster-state](../img/87-orchestration-cluster-state.png)
 
-- Data is archived in the background, coupled to the exporter but without blocking the exporter's progress.
-- Indices can be located in either Elasticsearch (ES) or OpenSearch (OS). Our web components (Tasklist and Operate) then use the new harmonized indices to show data to the user.
+In the past release, we’ve engineered a new Camunda Exporter, which brings the importer and archiving logic of web components (Tasklist and Operate) closer to our distributed platform (Zeebe). This allows us to simplify our installation, enable scalability for web apps, reduce the latency to show runtime and historical data, and reduce data duplication (resource consumption).
 
-The following diagram shows a simplified version of this work.
+![brown-field-without-optimize](../img/brown-field-orchestration-cluster-without-optimize.png)
 
-![Camunda Exporter diagram](../img/target-camunda-exporter.png)
-
-- For example, Tasklist and Operate Importers are still required for old data to be imported, but the Camunda exporter writes all new data into ES/OS. After old indices are drained, importers can be turned off.
-- The archiver, which takes care of the archiving of completed process instances, will be moved into the Zeebe system as well, to reduce the installation complexity and provide a better scaling and replication factor (based on partitions).
-- This helps achieve a streamlined architecture, and improves platform performance and stability (especially regarding ES/OS).
-- A new separate component covers the migration, which will be part of the single application but can also deployed separately. It will adjust the previous Operate indices to make them more harmonized and usable by Tasklist.
+The new Camunda Exporter helps us achieve a more streamlined architecture, better performance, and stability (especially concerning ES/OS).
+For more details about this project, you can also read our blog post [here](https://camunda.com/blog/2025/02/one-exporter-to-rule-them-all-exploring-camunda-exporter/).
 
 ### Harmonized index schema
 
-Camunda is harmonizing our index structure and usage.
+Our existing data schema in the secondary storage has been harmonized, to be used by all our components.
```

Comment:
Do users really know what harmonized indices mean? I feel like we should be a bit more explicit in saying exactly what we are referring to here

Reply:
This terminology is only used for the announcement page, to describe also the process of us harmonizing the indices, afterwards we don't want to use it anymore